frontend/online-shop/src/services/ProductService.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+class ProductService {
+  props: any;
+  constructor() {}
+
+  getProducts() {
+    const result = axios.get("http://localhost:8080/products", {});
```

Comment:
The second parameter is optional, you can just place the path as the first parameter

Reply:
Fixed